CondFormats/Luminosity/interface/LumiCorrections.h

```diff
@@ -0,0 +1,36 @@
+#ifndef LUMICORRECTIONS_H
```

Comment:
hi @capalmer85 - please add the subsystem and package name to the include guard.